workflow-examples/src/test/java/com/redhat/parodos/examples/integration/utils/ExamplesUtils.java

```diff
@@ -0,0 +1,27 @@
+package com.redhat.parodos.examples.integration.utils;
+
+import com.redhat.parodos.sdk.api.ApiException;
+import com.redhat.parodos.sdk.api.ApiResponse;
+import com.redhat.parodos.sdk.api.ProjectApi;
+import com.redhat.parodos.sdk.model.ProjectResponseDTO;
+import org.apache.http.HttpStatus;
+
+import java.util.List;
+
+/**
+ * @author Gloria Ciavarrini (Github: gciavarrini)
+ */
+public final class ExamplesUtils {
+
+	public static void waitProjectStart(ProjectApi projectApi) throws ApiException {
+		ApiResponse<List<ProjectResponseDTO>> projectsWithHttpInfo = projectApi.getProjectsWithHttpInfo();
+		for (int i = 0; i < 100; i++) {
```

Comment:
What do you think about sleeping instead of busy waiting? Maybe there is async way of doing it?

Reply:
Forgot to add a `sleep` inside the for loop. 

By the way, I was not sure on how to implement this. There is the possibility to use [spring-retry](https://www.baeldung.com/spring-retry) but I'm not sure if this is "too much" for an example package.

What do you think?

Reply:
There is a way to use `OkHttpClient` in async way. It is a client used internally by `ApiClient` and there is a `getHttpClient` method. Here is an [example](https://stackoverflow.com/questions/33418232/okhttp-update-ui-from-enqueue-callback#33427922) how to do it.

Reply:
Addressed. Can you please take a look? I also implemented a retry mechanism similar to the one available in the bash script but I'm not completely sure about it.